framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -636,6 +636,16 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
     }
 
     @Override
+    public List<AnnotationMirror> getJaifInferenceIgnoredAnnotations() {
+        List<AnnotationMirror> out = super.getJaifInferenceIgnoredAnnotations();
+        out.add(AnnotationUtils.fromClass(analysis.getEnv().getElementUtils(),
+                UnknownVal.class));
```

Comment:
I removed that method. Now, to check if an annotation should be ignored or not, the @ Target of the annotation is verified.